<?xml version="1.0"?>
<launch>
    <arg name="camera_topic" default="/usb_cam/image_raw"/>

    <node name="$(anon decoder)" pkg="sirio_utilities" type="decode_camera.py" output="screen">
        <!-- Pass the topic names as parameters -->
        <param name="compressed_topic" value="$(arg camera_topic)/compressed_encoder"/>
        <param name="raw_topic" value="$(arg camera_topic)/decoded_image_raw"/>
    </node>
</launch>
